doc/api/readline.md

```diff
@@ -703,14 +703,21 @@ added: v17.0.0
 The `readlinePromises.createInterface()` method creates a new `readlinePromises.Interface`
 instance.
 
-```js
-const readlinePromises = require('node:readline/promises');
-const rl = readlinePromises.createInterface({
-  input: process.stdin,
-  output: process.stdout,
+```mjs
+import { createInterface } from 'node:readline/promises';
+import { stdin, stdout } from 'node:process';
+const rl = createInterface({
+  input: stdin,
+  output: stdout,
 });
 ```
 
+```cjs
+const { createInterface } = require('node:readline/promises');
+const { stdin: input, stdout: output } = require('node:process');
+const rl = createInterface({ input, output });
```

Comment:
This example and the MJS example should be as close to identical as possible, except for the import.

Reply:
In other words, the examples should look exactly the same (except the import + process). That way, users can ensure consistency

Reply:
Done!

Thank you for taking the time to review the changes and give feedback to me :pray: !